Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_11.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Fixed an issue so that EML files downloaded from ***ews-get-items-as-eml*** command and **fetch_emails_as_incidents** will now retain CRLF line separators.
```

Comment:
```suggestion
- Fixed an issue so that EML files downloaded from the ***ews-get-items-as-eml*** command and **fetch_emails_as_incidents** will now retain CRLF line separators.
```